frontend/bower.json

```diff
@@ -33,8 +34,6 @@
     }
   },
   "resolutions": {
```

Comment:
I think that you can remove resolutions object. Please try remove it, and test app without resolutions.